bundles/net.tourbook.device/src/net/tourbook/device/sporttracks/FitLogSAXHandler.java

```diff
@@ -337,15 +337,14 @@ private void finalizeTour() {
          final StringBuilder tourNotes = new StringBuilder(tourData.getTourDescription());
 
          if (!tourNotes.toString().trim().isEmpty()) {
-            tourNotes.append(System.getProperty("line.separator")); //$NON-NLS-1$
-            tourNotes.append(System.getProperty("line.separator")); //$NON-NLS-1$
+            tourNotes.append(UI.NEW_LINE2);
          }
 
          tourNotes.append(Messages.FitLog_CustomDataFields_Label);
          _currentActivity.customDataFields.forEach((key, value) -> {
             if (!tourNotes.toString().trim().isEmpty()) {
                //If there is already content in the notes fields, then we insert a new line
-               tourNotes.append(System.getProperty("line.separator")); //$NON-NLS-1$
+               tourNotes.append(UI.SYSTEM_NEW_LINE);
```

Comment:
You should use `UI.SYSTEM_NEW_LINE` only when necesary, e.g. export txt files in other cases `net.tourbook.common.UI.NEW_LINE` works for win & linux in the UI